crates/jstz_cli/src/term/browser.rs

```diff
@@ -0,0 +1,32 @@
+use std::process::{Command, Stdio};
+
+use log::info;
+
+use crate::{error::Result, term::styles};
+
+pub fn open_browser(url: &str) -> Result<()> {
+    // TODO: Support Windows
+    if cfg!(target_os = "linux") {
+        let linux_cmd = format!(r#"xdg-open "{}""#, url);
+        Command::new("sh")
+            .arg("-c")
+            .arg(&linux_cmd)
+            .stdout(Stdio::null())
+            .stderr(Stdio::null())
+            .spawn()?;
+    } else {
+        let mac_cmd = format!(r#"open "{}""#, url);
+        Command::new("sh")
+            .arg("-c")
+            .arg(&mac_cmd)
+            .stdout(Stdio::null())
+            .stderr(Stdio::null())
+            .spawn()?;
+    };
+
+    info!(
+        "Opened a link in your default browser: {}",
```

Comment:
this is just a log, not something that gets printed in terminal, right?

Reply:
No, this gets printed in the terminal

Reply:
Like:
```sh
$ jstz docs
Opened a link in your default browser: https://trilitech.github.io/jstz/
```

Reply:
oh ok. I would prefer more not printing anything or atleast having a slightly simpler message e.g.:
"Opened docs - https://trilitech.github.io/jstz/ in your default browser."

Reply:
Modulo a 'docs', the messages are basically identical. The message is more generic to permit different links being opened in the future (e.g. wallets)